auditwheel/repair.py

```diff
@@ -10,6 +10,8 @@
 from typing import Dict, Optional
 import logging
 
+from wheel.install import WHEEL_INFO_RE  # type: ignore
```

Comment:
As wheel doesn’t have a public API this might break in the future. See https://github.com/pypa/wheel/issues/235#issuecomment-383713646